hmy/api_backend.go

```diff
@@ -206,3 +209,17 @@ func (b *APIBackend) GetBalance(address common.Address) (*hexutil.Big, error) {
 func (b *APIBackend) NetVersion() uint64 {
 	return b.hmy.NetVersion()
 }
+
+// GetEVM returns a new EVM entity
+func (b *APIBackend) GetEVM(ctx context.Context, msg core.Message, state *state.DB, header *types.Header) (*vm.EVM, func() error, error) {
+	state.SetBalance(msg.From(), math.MaxBig256)
```

Comment:
is it from ethereum code ? why do we need to set balance in state for From ?

Reply:
Yes. I'm not sure though. https://github.com/ethereum/go-ethereum/blob/40cdcf8c47ff094775aca08fd5d94051f9cf1dbb/les/api_backend.go#L114.

Posted a [question](https://ethereum.stackexchange.com/q/72977/54923)

Reply:
The code is borrowed from [go-ethereum](https://github.com/ethereum/go-ethereum/blob/40cdcf8c47ff094775aca08fd5d94051f9cf1dbb/les/api_backend.go#L114)